csi/node_server.go

```diff
@@ -824,6 +819,9 @@ func (ns *NodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 			return nil, status.Errorf(codes.FailedPrecondition, "volume %s requires shared access but is not marked for shared use", volumeID)
 		}
 
+		// For non-migratable RWX volumes with the volume mode `Block`, it would still be formatted with a filesystem and attached via NFS.
```

Comment:
> For non-migratable RWX volumes with the volume mode `Block`

The statement is not correct. Non-migratable RWX volumes doesn't have `Block` volume mode